compiler/qsc_qasm/src/stdlib/QasmStd/src/QasmStd/Angle.qs

```diff
@@ -1,6 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+/// This module defines the Angle type and its associated functions.
```

Comment:
I don't think we use 'module' to identify a piece of code, but I don't remember what we settled upon. It was either library or project...